index.ts

```diff
@@ -80,7 +68,7 @@ export class TypedMap<K, V> {
   entries: Array<TypedMapEntry<K, V>>
 
   constructor() {
-    this.entries = new Array<TypedMapEntry<K, V>>()
+    this.entries = new Array<TypedMapEntry<K, V>>(0)
```

Comment:
What does the `0` do?

Reply:
It creates an array of size 0. The TypeScript linter was complaining about this, although it was compiling fine before.